components/fxa-client/Cargo.toml

```diff
@@ -24,6 +24,10 @@ viaduct = { path = "../viaduct" }
 rc_crypto = { path = "../support/rc_crypto", features = ["ece", "hawk"] }
 error-support = { path = "../support/error" }
 thiserror = "1.0"
+anyhow = "1.0"
+pbkdf2 = "0.3.0"
+sha2 = "0.8.1" # pbkdf2 dep
+hmac = "0.7.1" # pbkdf2 dep
```

Comment:
How you would feel about implementing these operations in `rc_crypto`? (ofc I can mentor you here) I'm concerned that we'll be adding untrusted crypto code in mozilla-central.

Reply:
issue here https://github.com/mozilla/application-services/issues/3190

Reply:
I would love to 😃 I'll get started on it on a separate pr and will let you know if I have any questions as I go!